src/org/zaproxy/zap/extension/spider/OptionsSpiderPanel.java

```diff
@@ -89,6 +90,7 @@
 	private JCheckBox chkAcceptCookies;
 	private DomainsAlwaysInScopeMultipleOptionsPanel domainsAlwaysInScopePanel;
 	private DomainsAlwaysInScopeTableModel domainsAlwaysInScopeTableModel;
+	private ZapTextArea irrelevantUrlParameters;

```

Comment:
We have been moving away from free text options, as they are not as flexible as they should. For example, in this case I was expecting that the user could allow to specify a regular expression not just an exact string (it would also allow to remove the hardcoded "utm_" check).

Reply:
So, if I understand correctly there is no problem in using `ZapTextArea`, but I need to include interpreting its text as regexp, don't I?

Reply:
Technically there's no problem but it would be better (usability) if we didn't. @psiinon @kingthorin wdyt?

We can also handle the UI changes.

Reply:
Agree that we should really be using the newer control which allows the user to add, change and delete items. But if that does look too hard then as @thc202 said we can make that change.

Reply:
Ok, I've understood. Please, let me do it myself)

Reply:
@AleksandrPea further changes coming?

Reply:
Being done in zaproxy/zap-extensions#3991.